### DISCUSSION TITLE
Docker container start-dev vs start

### DISCUSSION THREAD

**Discussion by davehardyuk** [keycloak/keycloak#16271] [Q&A]
I'm using the Dockerfile on this page as a base: https://www.keycloak.org/server/containers running version 20.0.2

When running start-dev with no options, it's running fine, however when running just the "start" command, the site fails to load and a js error appears: 

`Uncaught (in promise) {error: 'Timeout when waiting for 3rd party check iframe message.'}`

Any ideas as to how I can combat that? I'm deploying the Docker image via AWS CDK

**Comment by tomasnorre** [comment]
I'm not sure. But I think this might be related to the template/theme changes. 
https://www.keycloak.org/docs/latest/upgrading/index.html#_migrate_themes 
https://www.keycloak.org/docs/latest/upgrading/index.html#migrating-to-19-0-0

Does this also happens if you set the default theme to keycloak?
